README.md

```diff
@@ -50,50 +92,27 @@ The AutoLoginFactory defines the following listener options:
 
   [SecurityBundle documentation]: http://symfony.com/doc/current/book/security.html#using-multiple-user-providers
 
-### Security Configuration Examples
-
-Consider the following example, which uses a stock EntityUserProvider:
-
-```yml
-services:
-    acme.auto_login_user_provider:
-        # Assume this class implements Jmikola\AutoLogin\User\AutoLoginUserProviderInterface
-        class: Acme\UserBundle\Security\AutoLoginUserProvider
-
-security:
-    providers:
-        acme_user_provider:
-            entity: { class: AcmeUserBundle:User, property: username }
-    firewalls:
-        main:
-            # We need not specify a "provider" for our firewall or listeners,
-            # since SecurityBundle will default to the first provider defined.
-            jmikola_auto_login:
-                auto_login_user_provider: acme.auto_login_user_provider
-                token_param: al
-```
+### An other security configuration example
 
-In this example, we customized the token's query parameter. We also needed to
-specify a custom service for `auto_login_user_provider`, since
-EntityUserProvider does not implement AutoLoginUserProviderInterface. We could
-simplify our configuration by using a custom service for our user provider,
-which implements both interfaces:
+In this example we specify a provider that implement both `UserProviderInterface` and `AutoLoginUserProviderInterface`. We do 
+also set the URL token to listen for to `auto_login`.
```

Comment:
"In this example, we specify that provider that implements both `UserProviderInterface` and `AutoLoginUserProviderInterface`. We also customize the URL parameter to use `auto_login` instead of the default."